### ISSUE TITLE
WSS freeze in FS 1.10.6

### ISSUE THREAD

**Issue by fkakiman** [signalwire/freeswitch#1920]
hello
I am using FS 1.10.6 with the latest sofia library and since a few days,
WSS freezes working and all the connections are closed.

suddenly, all the users registered are quicked off, and if i do a netstat,
a see a lot of connection for port 7443 with "Recv-Q" counter greater than
0 and in CLOSE_WAIT status.

all the tries for registering are not OK, but sofia profiles are UP and
running.

Doing a reload of mod_sofia doesn't work, it says that module is in use,
and we could only restart FS service in order to have it work again.

It happens several times a week but never at the same time and even when there is a few active registrations

**Comment by 2954456878** [comment]
Try using ws

**Comment by seven1240** [comment]
what about 1.10.8 and the latest sofia-sip?

**Comment by fkakiman** [comment]
> Try using ws

yes it is a good idea to try with WS, but is there a product to handle WSS and to convert and send in WS to FS ?

**Comment by 2954456878** [comment]
> > 尝试使用ws
> 
> 是的，尝试使用WS是一个好主意，但是有没有一个产品可以处理WSS，将WS转换并发送到FS？

nginx   OpenSIPS

**Comment by themsley-voiceflex** [comment]
This can probably be closed, duplicate of #2283
